docker-compose.yml

```diff
@@ -1,3 +1,5 @@
+version: "3.7"
```

Comment:
W nowych docker-compose nie trzeba ustawiać wersji: https://docs.docker.com/compose/compose-file/compose-versioning/

Reply:
U mnie nie chciało wstać, bo nie rozpoznawało poleceń :)

Reply:
A zerkniesz, czy to kwestia linuxa? Bo to jedynie sie rozni XD